src/bin/pgcopydb/pgcmd.c

```diff
@@ -448,12 +453,6 @@ pg_dump_db(PostgresPaths *pgPaths,
 		args[argsIndex++] = nspname;
 	}
 
-	args[argsIndex++] = "--file";
-	args[argsIndex++] = (char *) filename;
-	args[argsIndex++] = (char *) connStrings->safeSourcePGURI.pguri;
-
-	args[argsIndex] = NULL;
-
```

Comment:
Can we keep that even if the compiler does not need it now? The call to `initialize_program` requires a NULL pointer to end the array, and if the `argsIndex` is up to `PG_CMD_MAX_ARG` I would rather have to debug the NULL assignment out of bounds situation before the call to `initialize_program` compare to the undefined behavior and a later seemingly random crash.

Reply:
the new change initializes the array with NULL values.